src/GovUk.Education.ExploreEducationStatistics.Public.Data.Model/Migrations/20240710081112_EES5329_CreatePublicDataAdminRole.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Diagnostics.CodeAnalysis;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace GovUk.Education.ExploreEducationStatistics.Public.Data.Model.Migrations;
+
+/// <inheritdoc />
+[ExcludeFromCodeCoverage]
+public partial class EES5329_CreatePublicDataAdminRole : Migration
+{
+    /// <inheritdoc />
+    protected override void Up(MigrationBuilder migrationBuilder)
+    {
+        // Grant privileges on objects created by this resource's database user to the 'public_data_admin' role.
```

Comment:
Is this a repeat of the code in InitialMigration?

Reply:
Yes it is, this one is the migration required to make this change in the Dev environment as it stands now, because the `InitialMigration` has already run there.

This changes was copied back to `IntiialMigration` so we don't forget it if we squash the migrations again and produce a new `IntiialMigration`.

If you clear the Postgres volume locally and allow all the migrations to run from scratch it does mean these grants get applied twice but I don't think that matters.

Reply:
This has been replaced now by another migration `EES5329_GrantPublicDataReadWriteRolePrivileges` , but yes it's repeating some code also added to `InitialMigration`. 

This migration is required to make the change in the Dev environment as it stands now, because `InitialMigration` has already run there, assuming that we're not immediately about to tear down that database and recreate it.

I copied the code back into `InitialMigration` so we don't forget it if we squash the migrations again and produce a new `InitialMigration`. I thought we would be quite likely to diff any new `InitialMigration` with the existing one and then spot this addition and copy it over.

Reply:
Makes sense, thanks!